Packs/PAN-OS/ReleaseNotes/1_10_1.md

```diff
@@ -0,0 +1,11 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Added 6 commands.
+  - ***pan-os-platform-get-global-counterss***
+  - ***pan-os-platform-get-bgp-peers***
+  - ***pan-os-platform-get-available-software***
+  - ***pan-os-platform-get-ha-state***
+  - ***pan-os-platform-get-jobs**
+  - ***pan-os-platform-download-software***
+- Updated the Docker image to: *demisto/pan-os-python:1.0.0.27793*.
```

Comment:
Remove it as you didn't really update the docker-image

Reply:
Fixed